app-doc/pytorch-docs/pytorch-docs-1.3.1.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 2019-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{5,6,7} )
```

Comment:
We are going rid of Python 2.7 and 3.5. Please drop them.  The same applies to all other ebuilds.

Reply:
Yes, I also planned this change after reading the news. Now fixed.